web/README.md

```diff
@@ -50,10 +71,9 @@ By being enabled, the USER_10 message at the set values is sent over the period
 provided in the input box (defaulted at 1000 ms). Modify the values concurrently by dragging the sliders
 to the desired ones.
 
-#### Note
-The values sent by the message change synchronously as the slider values
-are updated, even while sending is enabled. Nonetheless, the period over
-which the message is sent will not update if it's changed while sending is enabled.
+***Note:*** _The values sent by the message change synchronously as the slider values_
+_are updated, even while sending is enabled. Nonetheless, the period over_
+_which the message is sent will not update if it's changed while sending is enabled._
 
-In order to avoid unexpected behavior, if the client's connection is interrupted or lost,
-the server will be notified and all values will be set to 0 automatically.
+_In order to avoid unexpected behavior, if the client's connection is interrupted or lost,_
```

Comment:
FYI markdown automatically wraps text to the next line. I know 231 teaches you that 80 character line limit is the be-all end-all, but I've never seen it enforced or practiced in markdown. Usually, I write paragraphs all on one line and let the editor do that wrapping for me. It's easier that way for formatting like italics, bold, etc.